tests/test_imports.py

```diff
@@ -1,25 +1,28 @@
 # flake8: noqa
 
 import torch
-from hydra_configs.torch.utils.data import DataLoaderConf
 
 print(f"ENV_TORCH_VERSION: {torch.__version__}")
-print(f"Imported DataLoaderConf: {DataLoaderConf}")
 
+# Direct class import (preferred):
+from hydra_configs.torch.utils.data import DataLoaderConf
+
+print(f"Imported DataLoaderConf: {DataLoaderConf}")
```

Comment:
can you make this a real test?
1. Use local imports (inside the test functions).
2. assert that the class name is what you expect.

Reply:
I've removed this test for now. I don't think it will be relevant until we support the legacy imports as a feature. I'm going to put that on a separate 'experimental' branch and there can be a PR for it. I think its important to fix the structure and complete this PR to unblock finishing out the rest of the useful config files.